package NYT.NYTree.NProto;

////////////////////////////////////////////////////////////////////////////////

message TReqGet
{
    repeated bytes attributes = 1;
}

message TRspGet
{
    required bytes value = 1;
}

////////////////////////////////////////////////////////////////////////////////

message TReqSet
{
    required bytes value = 1;
}

message TRspSet
{ }

////////////////////////////////////////////////////////////////////////////////

message TReqRemove
{ }

message TRspRemove
{ }

////////////////////////////////////////////////////////////////////////////////

message TReqList
{ }

message TRspList
{
    required bytes keys = 1;
}

////////////////////////////////////////////////////////////////////////////////
